app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+    validates :name, presence: true
+    validates :email, uniqueness: true
+    has_secure_password
```

Comment:
dùng indent 2 space thôi em nha
```suggestion
  validates :name, presence: true
  validates :email, uniqueness: true
  has_secure_password
```

Reply:
dạ chị

Reply:
sao file này chưa fix cmt vậy e?

Reply:
ủa dạ, e có fix r chị! e sửa thành 1 tab là bằng 2 space như a Duy bày ạ!

Reply:
File này vẫn đang sai indent này b. Có thể do settings vscode chưa chuẩn nên tab size thay đổi khi push code

Reply:
Dạ, ac bày e sửa với được k á! e chưa hiểu chỗ này lắm ạ...

Reply:
Thử thêm 2 cái này vào settings vscode xem đc không nhé b. format lại file rồi push lại thử 
"editor.detectIndentation": false,
"editor.insertSpaces": false,

Reply:
dạ a! E cảm ơn ạ

Reply:
Ac xem hộ giùm e được không với ạ?